test/referenceData.js

```diff
@@ -19,35 +18,53 @@ var globalSuiteNoNesting = new Suite("", [groupA, groupB], []);
 // Global Suite for Jasmine
 var globalSuiteNesting = new Suite("", [groupA, groupB, parent], []);
 
+function toStartTest(test) {
+    return new Test(
+        test.testName,
+        test.suiteName,
+        undefined,
+        undefined,
+        undefined
```

Comment:
Explicitly passing undefined when there are no other defined arguments following seems pointless. Should just drop these three. Applies to similar usages above.